### DISCUSSION TITLE
Verification when Instution doesn't offer academic email addresses

### DISCUSSION THREAD

**Discussion by areaven** [community/community#57464] [GitHub Education]
### Hi everyone,

Hey! I signed up for Github Education in December last year, and back then it just asked me for documentation. Fast forward to a month ago and, when renewing, my university now requires an academic email (which we students don't have, it's only used for administration).
![firefox_zl6rK828q1](https://github.com/community/community/assets/8731625/6dbc9a5b-0d71-43b3-a41b-74aff1f260b5)
I've asked uni for some clarification or an academic email to no avail, since they really don't give addresses to students. What should my course of action be?

**Comment by herbe13** [comment]
I have the same problem with my university, but several email addresses appear in the list, but not from my specialization... those from GitHub must be notified, but I don't know how

**Comment by chriskthomas** [comment]
I ran into this issue when signing up as a teacher. I just typed the name of my school without selecting from this list. This freed me up to select any email address.

**Comment by samus-aran** [comment]
Hi @areaven  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
